### PR TITLE
Updates the forms to use the new form styling

### PR DESCRIPTION
**Pull request by floriank**
Also removes a good chunk of prototype based JavaScript functionality
and displays multiple checkouts simultaneously instead of accessing them
one after the other.

This was done to simplify the process as well as get rid of old
functionality that is not really compatible with the new form stuff.

Sould also fix https://community.openproject.org/work_packages/18685

### CODE REVIEW

File: app/views/settings/_checkout_scm.html.erb [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
+  <div class="form--field">
+    <%= setting_check_box "checkout_display_command_#{scm}", :label => :field_checkout_display_command %>
+  </div>
 
   <%= javascript_tag do %>
     <% repo = "Repository::#{scm}".constantize %>
     var subform = new Subform('<%= escape_javascript(render(:partial => "checkout_protocol", :locals => {:protocol => OpenProject::Checkout::Protocol.new({:protocol => scm, :append_path => (repo.allow_subtree_checkout? ? '1' : '0'), :command => repo.checkout_default_command}), :scm => scm})) %>',<%= Setting.send("checkout_protocols_#{scm}").length %>,'settings_checkout_protocols_<%= scm %>');
     protocolForms.set('<%= scm %>', subform);
-  <% end %>
+  <% end %>s
```
Comment (ulferts) [inline app/views/settings/_checkout_scm.html.erb]: that "s" is wrong here

**Comment by ulferts** [comment]
@floriank why did you remove the flash based "copy to clipboard" functionality?

**Comment by floriank** [comment]
Never worked for me - I could not deduce the functionality from the implementation given (because it was all over the place).

I can restore it - the previous styling is incompatible with the new styling though

**Comment by floriank** [comment]
my suggestion would be to make it a bit more obvious - like adding an affix (suffix?) field to the input displaying the repo url, containing a copy-icon, making the functionality more obvious.

**Comment by ulferts** [comment]
GitHub's solution is pretty obvious I guess: 
image

**Comment by floriank** [comment]
I'd like that.

**Comment by ulferts** [comment]
@floriank and I talked about it. I still had the state in mind where such a feature was existent. As it is not, and the scope of this one is just styling, it is good to merge.

**Pull request merged**